core/mm6Misc.js

```diff
@@ -115,6 +115,11 @@ CLASS({
         []), sink);
     }
   }*/
+  methods: [
+    function toE(X) {
```

Comment:
It actually works nicely that way. Someday we will want to override the view for a relationship.